src/main/java/app/service/impl/FruitParserImplements.java

```diff
@@ -0,0 +1,23 @@
+package app.service.impl;
+
+import app.model.SupplyFruit;
+import app.service.FruitParser;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+public class FruitParserImplements implements FruitParser {
```

Comment:
Maybe you meant FruitParserImplementation? think about it :)

Reply:
Sure...